python/ingestion/url_file_to_gcs.py

```diff
@@ -64,10 +76,23 @@ def download_first_url_to_gcs(url_list, gcs_bucket, dest_filename,
             dest_filename)
         return
 
-    # Download URL locally, upload to bucket and remove local file
-    local_path = local_file_path(dest_filename)
-    with file_from_url, open(local_path, 'wb') as f:
+    # Download the contents of the URL to a local file
+    url_local_path = local_file_path(dest_filename)
+    # Downloads the current file in GCS to a local file
+    gcs_local_path = local_file_path("gcs_local_file")
+    with file_from_url, open(url_local_path, 'wb') as f, open(gcs_local_path, "wb") as file_obj:
```

Comment:
optional nit: maybe rename file objects to new and old instead of f and file_obj?

Reply:
sgtm, done.